src/constants.ts

```diff
@@ -10,6 +10,7 @@ export const GLOBAL_CONF_VERSION = '1';
 export const REGIONS_API_URL: { [key: string]: string } = {
   iran: 'https://api.iran.liara.ir',
   germany: 'https://api.liara.ir',
+  objStorage: 'https://storage-service.iran.liara.ir',
```

Comment:
Please move this to a separate constant called `OBJECT_STORAGE_API_URL`. You can also add another constant for `OBJECT_STORAGE_API_URL_DEV`.